script/Ciel/Day 11/11_01_CIEL11_1.txt

```diff
@@ -11,12 +11,13 @@ A blue sky with huge, towering cumulonimbus clouds.
 [sha:c12e997c9c8593402cca860e0f4c8623cb0ab971]{
 -- Page 1, Offset 28973.
 -- じりじりとゆらぐ風景と
-The landscape shimmers in the heat
+-- TRANSLATION HERE
```

Comment:
Seems like an error, but we can solve that later. Assuming the rest of the edits are reexports.

Reply:
That's correct, my folder was missing some of the Ciel files. Shouldn't show up in future edits